testing/integration/kubernetes_agent_standalone_test.go

```diff
@@ -262,8 +267,8 @@ func deployK8SAgent(t *testing.T, ctx context.Context, client klient.Client, obj
 	command := []string{"elastic-agent", "status"}
 	var stdout, stderr bytes.Buffer
 	var agentHealthyErr error
-	// we will wait maximum 60 seconds for the agent to report healthy
-	for i := 0; i < 60; i++ {
+	// we will wait maximum 90 seconds for the agent to report healthy
+	for i := 0; i < 90; i++ {
```

Comment:
90 seconds still seems tight, how about 120 seconds?

Reply:
sure thing, "better safe than sorry" is my policy in this time checks 🥲 pushing